parts/1-intro/slides.md

```diff
@@ -0,0 +1,103 @@
+# Introduction
+
+## 0. Brief introduction to genomics
+Compare DNA and other concepts to programming!
+
+1. The DNA is byte code but instead of 0s and 1s we have As, Cs, Ts, and Gs.
+
+2. So what do they encode? Well combining a streach of bases you get genes. A gene
+   encodes a protein: very same you eat to build muscles. They usually have one very
+   specific task to carry out in the body.
+
+3. Genes are further grouped into longer streaches of DNA strands called chromosomes
+   which make up the software for your body.
+
+4. Lastly this code is run inside the cell which is the hardware that understands what
+   to make of the code. Basically you can a cell and "boot it up" with whatever DNA
```

Comment:
some word is missing between "can" and "a cell", right?

I like the analogies!